src/database/initialization/organization_proposal_to_json.sql

```diff
@@ -0,0 +1,26 @@
+CREATE OR REPLACE FUNCTION organization_proposal_to_json(organization_proposal organizations_proposals)
+RETURNS JSONB AS $$
+DECLARE
+  proposal_json JSONB;
+  organization_json JSONB;
+BEGIN
+  SELECT proposal_to_json(proposals.*)
+  INTO proposal_json
+  FROM proposals
+  WHERE proposals.id = organization_proposal.proposal_id;
+
+  SELECT organization_to_json(organizations.*)
+  INTO organization_json
+  FROM organizations
+  WHERE organizations.id = organization_proposal.organization_id;
+
+  RETURN jsonb_build_object(
+    'id', organization_proposal.id,
+    'organizationId', organization_proposal.organization_id,
+    'organization', organization_json,
+    'proposalId', organization_proposal.proposal_id,
+    'proposal', proposal_json,
```

Comment:
Is this a 1:1 replationship? I knew we weren't doing many:many at this point, but I did assume 1:many. The singular naming of the `proposalId`/`proposal` structure makes me doubt this though.

Reply:
this entity represents an individual link between a proposal and an organization (it's an OrganizationProposal).

An organization proposal will have exactly 1 organization and 1 proposal
A given organization can have any number of organization proposals
A given proposal can have any number of organiation proposals